Framework/PythonInterface/mantid/plots/__init__.py

```diff
@@ -43,6 +43,8 @@ class MantidAxes(Axes):
     '''
 
     name = 'mantid'
+    HORIZONTAL = BIN = 0
```

Comment:
Are these used anywhere?

Reply:
[Yes](https://github.com/mantidproject/mantid/pull/23987/files#diff-85780f399152791d6f31e1f45d2bd1f8R133), however I don't think `MantidAxes` is the best place for them. I would like to move them into an Enum and reuse it in `helperfunctions` (currently it gives an import error as it is a circular import) in order to remove `0` and `1` that are hard to understand at first glance.

Do you have a suggestion as to where it might be a good place for such an enum, so that it is importable in `helperfunctions`, but also in the `widgets`?

Reply:
I'm not sure how to avoid the circular import in this case - I've had similar problems in the past with defining defaults for functions like this and I have not come to a solid conclusion. One option would be a completely separate module containing this definition?